xla/stream_executor/command_buffer.h

```diff
@@ -64,14 +64,16 @@ class CommandBuffer {
   //
   //   (1) kCreate:    a new command buffer under construction
   //   (2) kUpdate:    updating a previously finalized command buffer
-  //   (3) kFinalized: command buffer ready for execution
+  //   (3) kFinalized: command buffer ready for execution, can be updated
```

Comment:
@anlunx Having the work Finalized, but that it can be updated sound very strange.
Should we change the name?
Can we limit what can be updated to a small set, like the ptr?